modules/pruning-images.adoc

```diff
@@ -28,20 +28,39 @@ By default, the integrated registry caches blobs metadata to reduce the number
 of requests to storage, and increase the speed of processing the request.
 Pruning does not update the integrated registry cache. Images pushed after
 pruning that contain pruned layers will be broken, because the pruned layers
-that have metadata in the cache will not be pushed. Therefore, it is necessary
-to clear the cache after pruning. This can be accomplished by redeploying the
-registry:
+that have metadata in the cache will not be pushed. To avoid this issue,
+you can set the image registry to `readOnly: true` before pruning and 
+reset to `false` when complete.
 
 ----
-$ oc rollout latest dc/docker-registry
```

Comment:
nit: do we want to remove this for master?  I thought this would work, and still be preferred, for 4.3+.  But I'm unfamiliar with the registry internals.

Reply:
@wking this was from 3.x. OpenShift 4 has completely different namings for the deployment

Reply:
@wking I was planning to merge this PR to master and cherrypick to 4.1 and 4.2 for now. Then merge https://github.com/openshift/openshift-docs/pull/17966/files to master when 4.3 is ready to go. The 4.3 PR has: `$ oc rollout restart deployment/image-registry -n openshift-image-registry`